src/ocamlorg_frontend/pages/learn.eml

```diff
@@ -101,7 +104,7 @@ Learn_layout.single_column_layout
     </div>
   </div>
 
-  <div class="section-mild-contrast">
+  <div class="section-mild-contrast dark:bg-none dark:bg-gradient-to-r dark:from-[#010105] dark:to-[#10162D] dark:border-none">
```

Comment:
Styles should be added to the existing `section-mild-contrast` class (see `bg_sections.css`), if Tailwind allows that. If Tailwind doesn't allow that, introduce a `dark:section-mild-contrast` class.

Reply:
Hi @sabine  i am not sure how to apply these style in `bg_sections.css` i tried this https://github.com/oyenuga17/ocaml.org/blob/db689764c9cf2802241f4710bca2043a9fa0973a/src/ocamlorg_frontend/css/partials/bg_sections.css#L8-L11

Reply:
When you do it like this in `bg_sections.css`, it works:

```
.dark .section-mild-contrast {
  background: linear-gradient(45deg, #010105, #10162D);
  @apply border-none;
}
```